advocacy_docs/migrating/oracle/migration_journey.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: The database migration “journey”
+---
+
+Migrating your database consists of a nine-step “journey.”
+
+![Migration journey](images/journey.png)
+
+## 1. Decide to migrate
+
+First, your organization must make the decision to migrate. You might decide you're spending too much money on your legacy database system. You might have initiatives that your current database technology doesn't support well. Or maybe you want to move to the cloud or take advantage of open-source software. These are common factors that might lead to your business case for migrating.
+
+Once you have your business case and your organization buys into it, your organization must commit to performing the migration. Commitment includes funding and resources to perform the migration.
+
+## 2. Analyze feasibility and alternatives
+
+Analyze your alternatives and look at the feasibility of migrating to certain databases. 
+
+### Review your application portfolio to determine whether all of your applications will migrate 
+
+As part of this review, you'll look at the applications that have legacy databases that you can and want to migrate from. Align your migration priorities with IT strategies to ensure these are important enough business applications to consider for migration. 
+
+As part of reviewing your application portfolio, you might decide that you can retire some applications. There might be other applications in your portfolio that you want to migrate but can't because they're supported only on the existing database. 
+
+As you're identifying a candidate set of applications to migrate, keep in mind that the desire to migrate them must correspond to an organizational commitment to completing the migration. It will take time, resources, and budget. 
+
+### Evaluate which database technologies are the right choice as the migration target for each of your applications 
+
+In many cases, EDB Postgres Advanced Server is an ideal choice to replace Oracle. However, some applications have requirements that might be better addressed by other solutions. 
+
+After selecting the appropriate set of target database technologies, a follow-on step is to perform a more detailed assessment of how complex the migration will likely be. When EDB Postgres Advanced Server is a potential target, you can use the EDB Migration Portal to assess how compatible the Oracle schema and procedural objects in the database are with EDB Postgres Advanced Server. This information is important in the next step of the journey.
```

Comment:
I assume that in an "ideal world" there's a higher-level software tool that can help the earlier phase of the process, but from where we stand today this reads a bit like putting the cart before the horse - presumably the output of the Migration Portal would be useful input for making the decision to use EPAS, rather than something only applied _after_ that decision is made?